### ISSUE TITLE
Fatal error: Cannot redeclare class simple_html_dom_node

### ISSUE THREAD

**Issue by Mictantecutli** [woocommerce/woocommerce#14808]
### EXPLANATION OF THE ISSUE

Hi,
I get 
Fatal error: Cannot redeclare class simple_html_dom_node in /home/mictantecuob/ftp/dietta/wp-content/plugins/woocommerce-dotpay/vendor/simple_html_dom.php on line 115
when I'm trying to activate "WooCommerce Dotpay Gateway" 3.0.8 version on WooCommerce 3.0.5 version.
I tried older versions of Dotpay with no result.
Before I had WP e-commerce and dotpay and everything was alright. 

### SYSTEM STATUS

<details>
```
`
### WordPress Environment ###

Home URL: http://www.dietta.pl
Site URL: http://www.dietta.pl
WC Version: 3.0.5
Log Directory Writable: ✔
WP Version: 4.7.3
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: pl_PL

### Server Environment ###

Server Info: Apache/2
PHP Version: 5.6.30
PHP Post Max Size: 64 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.47.0
OpenSSL/1.0.2g

SUHOSIN Installed: –
Max Upload Size: 64 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.0.5
WC Database Prefix: wp_
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔

### Security ###

Secure connection (HTTPS): ❌Twój sklep nie używa protokołu HTTPS. Dowiedz się więcej o HTTPS i certyfikatach SSL .
Hide errors from visitors: ✔

### Active Plugins (27) ###

Advanced CSS Editor: przez Hardeep Asrani – 3.0
Contact Form Generator: przez Creative Solutions – 2.1.7
Content Views: przez PT Guy – 1.9.4
Cookie Law Info: przez Richard Ashby – 1.5.3
Cryout Serious Theme Settings: przez Cryout Creations – 0.5.7
Custom Meta Widget: przez bitacre – 1.4.8
Duplicate Post: przez Enrico Battocchi – 3.1.2
Easy Facebook Likebox: przez Sajid Javed – 4.3.0
Image Hover Effects Ultimate Pro: przez Biplob Adhikari – 6.2
MaxButtons: przez Max Foundry – 6.12
Max Mega Menu: przez Tom Hemsley – 2.3.5
OTF Regenerate Thumbnails: przez Benjamin Intal - Gambit Technologies Inc – 0.3
Portfolio Gallery: przez Gallery By Huge IT – 2.3.3
Post Slider WD: przez WebDorado – 1.0.30
Shortcode Widget: przez Gagan Deep Singh – 1.2
Page Builder by SiteOrigin: przez SiteOrigin – 2.4.21
SiteOrigin Widgets Bundle: przez SiteOrigin – 1.7.2
TinyMCE Advanced: przez Andrew Ozz – 4.4.3
TR Post Slider Widget: przez Mohammad Tanzilur Rahman – 3.0
W4 Post List: przez Shazzad Hossain Khan – 2.1.2
Wise Chat: przez Marcin Ławrowski – 2.5
WooCommerce: przez Automattic – 3.0.5
WP Blog and Widget: przez WP Online Support – 1.3
WP Multibyte Patch: przez Seisuke Kuraishi – 2.4
Slimstat Analytics: przez Jason Crouse – 4.6.4
WP Statistics: przez Greg Ross & Mostafa Soufi – 12.0.5
WP Ultimate Post Grid: przez Bootstrapped Ventures – 2.3.1

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: PLN (zł)
Currency Position: right_space
Thousand Separator: 
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Strona sklepu: #883 - /?page_id=883
Koszyk: #884 - /?page_id=884
Zamówienie: #885 - /?page_id=885
Moje konto: #886 - /?page_id=886

### Theme ###

Name: Nirvana
Version: 1.3.3
Author URL: https://www.cryoutcreations.eu
Child Theme: ❌ – Jeśli modyfikujesz WooCommerce lub motyw nadrzędny
którego nie jesteś autorem
polecamy używanie motywu potomnego. Zobacz: Jak tworzyć motywy potomne

WooCommerce Support: ✔

### Templates ###

Archive Template: Twój motyw zawiera plik woocommerce.php
nie możesz zmodyfikować szablonu woocommerce/archive-product.php
ponieważ woocommerce.php ma wyższy priorytet niż archive-product.php. Ma to na celu zapobieganie problemów z wyświetlaniem.

Overrides: –
`
```
</details>

**Comment by claudiulodro** [comment]
That error is coming out of the Dotpay gateway. We try to keep things backwards compatible with new WooCommerce releases as much as possible, but plugin developers have to update their plugins occasionally. It looks like Dotpay has released [version 3.0.9](https://github.com/dotpay/WooCommerce2/releases/tag/v3.0.9). That might solve your problems.

**Comment by Mictantecutli** [comment]
Indeed. Thank You.
